Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-contracts.controller.ts

```diff
@@ -16,14 +16,14 @@
         ($scope, $http, itSystemUsage, entityMapper, uiState: Kitos.Models.UICustomization.ICustomizedModuleUI, apiUseCaseFactory: Kitos.Services.Generic.IApiUseCaseFactory, contractUiState: Kitos.Models.UICustomization.ICustomizedModuleUI, itSystemUsageService: Kitos.Services.ItSystemUsage.IItSystemUsageService) => {
             var usageId = itSystemUsage.id;
             bindContracts(itSystemUsage);
-            var currentMainContract = null;
+            var currentMainContract: number;
```

Comment:
the problem with POST was that the "bindContracts" method was being called before the null assignment, so first it set the value, and than changed the value to null so the "saveMainContract" condition was never true